templates/custom-template/custom-template/flake.nix

```diff
@@ -1,46 +1,36 @@
 {
-  description = "Template for Holochain app development";
+  description = "Flake for Holochain app development";
 
   inputs = {
-    holochain-nix-versions.url  = "github:holochain/holochain?dir=versions/weekly";
+    holonix.url = "github:holochain/holonix?ref=main-0.3";
 
-    holochain-flake = {
-      url = "github:holochain/holochain";
-      inputs.versions.follows = "holochain-nix-versions";
-    };
-
-    nixpkgs.follows = "holochain-flake/nixpkgs";
-    flake-parts.follows = "holochain-flake/flake-parts";
-    scaffolding.url = "github:holochain/scaffolding";
+    nixpkgs.follows = "holonix/nixpkgs";
+    flake-parts.follows = "holonix/flake-parts";
```

Comment:
This is correct here, we want consumers who aren't adding other things to their flakes to just be as likely as possible to hit the cache!